addons/l10n_ch/data/l10n_ch_chart_post_data.xml

```diff
@@ -8,5 +8,7 @@
         <field name="income_currency_exchange_account_id" ref="ch_coa_3806"/>
         <field name="expense_currency_exchange_account_id" ref="ch_coa_4906"/>
         <field name="default_pos_receivable_account_id" ref="ch_coa_1101" />
+        <field name="default_cash_difference_income_account_id" ref="ch_coa_4991"/>
+        <field name="default_cash_difference_expense_account_id" ref="ch_coa_4992"/>
```

Comment:
Shouldn't it be the opposite ? 4991 for  expense (loss), and 4992 for income (gain) ?

Reply:
Yep indeed - I got my wires crossed